### DISCUSSION TITLE
404 result from snapshot command with --base-url option

### DISCUSSION THREAD

**Discussion by sheakelly** [percy/cli#1037] [Support]
## The problem

I am taking snapshots of a directory of html files that are not hosted at the root path. When I set
the --base-url to be deeper that one level I get a 404 error from the percy cli

## Environment

- Node version: 14.1
- `@percy/cli` version: 1.8.1
- Version of Percy SDK you’re using:
- If needed, a build or snapshot ID:
- OS version: 
- Type of shell command-line [interface]: zsh

## Details

If necessary, describe the problem you have been experiencing in more detail.

## Debug logs

```
percy snapshot public/ --allowed-hostname images.ctfassets.net --base-url /services/aged-care --debug --verbose
[percy:config] Config file not found (0ms)
[percy:config] Using config:
{
  discovery: {
    allowedHostnames: [
      'images.ctfassets.net'
    ]
  }
} (19ms)
[percy:core:browser] Launching browser (19ms)
[percy:core:browser] Browser connected [76445]: HeadlessChrome/96.0.4664.0 (243ms)
[percy:core] Percy has started! (0ms)
[percy:core] Stopping percy... (33ms)
[percy:core:browser] Closing browser (1ms)
[percy:core:browser] Browser closed (13ms)
[percy:core] Build not created (0ms)
[percy:cli] Error: 404 Not Found
    at handleFinished (file:///Users/sheakelly/Code/lwb/www-lwb-org-au/node_modules/@percy/client/dist/utils.js:176:17)
    at IncomingMessage.<anonymous> (file:///Users/sheakelly/Code/lwb/www-lwb-org-au/node_modules/@percy/client/dist/utils.js:193:27)
    at IncomingMessage.emit (events.js:412:35)
    at endReadableNT (internal/streams/readable.js:1334:12)
    at processTicksAndRejections (internal/process/task_queues.js:82:21) (0m
```

**Comment by harsh-browserstack** [comment]
Hi,

I do understand your concern and would kindly request you to reach out to Percy Support using our Contact Us page with more details such as the Percy CLI logs along with the debug logs.

The debug logs can be generated by exporting the below environment prior to the test:
​
$ export LOG_LEVEL=debug PERCY_DEBUG=*

The logs are generated in the terminal itself. Copy the same and paste it into a text file and I'll take a look at it.

**Reply by wwilsman** [reply]
@harsh-browserstack the debug logs are actually already in the submitted issue

**Comment by wwilsman** [comment]
Hey @sheakelly!

This does appear like it could potentially be a bug. I'll take a look at this today to know for sure and get back to you. If it is a bug, I'm sure I can ship a fix for you pretty quickly. 👍 

**Update**: This is indeed a bug, and the fix is here: #1042. This will likely ship tomorrow or after the weekend.

**Reply by sheakelly** [reply]
This is great news @wwilsman. Really appreciate the fast feedback here :)
